assets/keymaps/textmate.json

```diff
@@ -39,13 +39,13 @@
       "cmd-shift-left": "editor::SelectToBeginningOfLine",
       "cmd-shift-right": "editor::SelectToEndOfLine",
       "alt-shift-left": [
-        "editor::SelectToBeginningOfLine",
+        "editor::SelectToPreviousSubwordStart",
         {
           "stop_at_soft_wraps": true
         }
       ],
       "alt-shift-right": [
-        "editor::SelectToEndOfLine",
+        "editor::SelectToNextSubwordEnd",
```

Comment:
Are you sure you want `Subword` here instead of `Word`?

Reply:
@mrnugget yes, that makes sense.

Updated here: https://github.com/zed-industries/zed/pull/10535/commits/c452d2a3dc4172622f7dcfb807bee2cf37ab4668